app/controllers/admin/rebate_forms_controller.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 class Admin::RebateFormsController < Admin::BaseController
-  before_action :set_rebate_form, only: %i[show update destroy]
+  before_action :set_rebate_form, only: %i[show update destroy edit]
```

Comment:
the `def edit; end` is missing... ?

Reply:
@Br3nda it worked without it but I've put it back.